.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: go
+
+go:
+  - 1.5
+  - tip
+
```

Comment:
I think the PR builds will still work with this being present since Travis picks up the branch containing the commit. By mentioning this, we will be white-listing the master. This approach is followed by many HashiCorp projects and it all works just as expected.